daemon/cluster/metrics.go

```diff
@@ -0,0 +1,23 @@
+package cluster
+
+import "github.com/docker/go-metrics"
+
+var (
+	clusterInfo     metrics.LabeledGauge
+	clusterNodes    metrics.Gauge
+	clusterManagers metrics.Gauge
+)
+
+func init() {
+	ns := metrics.NewNamespace("engine", "daemon", nil)
+
+	clusterInfo = ns.NewLabeledGauge("swarm", "The information related to the cluster", metrics.Unit("info"),
+		"cluster_id",
+		"node_id",
+		"node_addr",
+	)
+	clusterManagers = ns.NewGauge("cluster_managers", "The number of node managers the cluster has", metrics.Unit("managers"))
```

Comment:
I think they should; otherwise the engine would have to poll for them. In this PR they're updated every time `/info` is invoked, but that doesn't seem right.